### DISCUSSION TITLE
How to authenticate to management API?

### DISCUSSION THREAD

**Discussion by nazarewk** [netbirdio/netbird#588] [Q&A / Support]
I'm trying to make some automation for configuring routes and DNS, but I cannot find any resources on ways of authenticating to management APIs

related to:
- https://github.com/netbirdio/netbird/discussions/587
- https://github.com/netbirdio/netbird/discussions/599

**Comment by ghost** [comment]
Hey @nazarewk have you found a way to call the Netbird management API for automation purposes?

**Comment by braginini** [comment]
Guys, there is no straightforward way yet. theoretically you could use the tokens generated by the dashboard. But they expire often.

We have a public API access planned for the next month.
